source/nodejs/adaptivecards/src/strings.ts

```diff
@@ -34,4 +34,7 @@ export class Strings {
     static readonly hints = {
         dontUseWeightedAndStrecthedColumnsInSameSet: () => "It is not recommended to use weighted and stretched columns in the same ColumnSet, because in such a situation stretched columns will always get the minimum amount of space."
```

Comment:
Interesting - this string -- where are we using these hints again? (_am assuming these get localized by some magic?_)

Reply:
They don't currently get localized, but this is just set up to allow that. The hints are used in the designer code editor, I believe.

Reply:
@shalinijoshi19 the point of these resource variables is to allow consuming applications to localize themselves. We do not ship localized SDKs.